dashboard/src/modules/components/ToastNotificationComponent/index.jsx

```diff
@@ -33,7 +33,10 @@ const ToastComponent = () => {
             />
           }
         >
-          {item?.message && <p>{item?.message}</p>}
+          {item?.message &&
+            item?.message.split("\n").map((i, key) => {
```

Comment:
We don't need (and therefore shouldn't use) the `?.` operator in the second reference.